src/reducers/match.js

```diff
@@ -0,0 +1,23 @@
+const initialState = {  
```

Comment:
right now this reducer doesn't do anything, but I created it because it'll be populated with match-related data pulled from the API later